attributes/default.rb

```diff
@@ -18,100 +18,100 @@
 #
 
 # cluster identifier
-default[:mongodb][:client_roles] = []
-default[:mongodb][:cluster_name] = nil
-default[:mongodb][:shard_name] = 'default'
+default['mongodb']['client_roles'] = ['']
```

Comment:
I think you have accidentally changed an *empty array* with an *array with an empty string* here. I cannot see where is this attribute used, but just in case.